storage/s3.go

```diff
@@ -241,10 +243,12 @@ func (s *S3) Move(ctx context.Context, srcPath, dstPath string) error {
 	return nil
 }
 
-func (s *S3) List(ctx context.Context, prefix string) ([]string, error) {
+// List function list object contained in bucket up to 1,000 objects, if maxKeys > 1,000, S3 will set maxKeys to 1,000
```

Comment:
It seems that this is not documented but in practice that's what happen, is that better to return an error if `maxKeys>S3ListMaxKeys` or to clamp maxKeys to S3ListMaxKeys if `maxKeys>S3ListMaxKeys` ?

Reply:
I actually don't like that kind of assertion without a solid proof. But the solid proof is actually here ;)

> Returns some or all (up to 1,000) of the objects in a bucket with each request.

https://github.com/aws/aws-sdk-go-v2/blob/v1.17.1/service/s3/api_op_ListObjectsV2.go#L16

Reply:
I think it is. Translated in French it would mean:

> Renvoie une partie ou la totalité (jusqu'à 1 000) des objets d'un bucket à chaque requête.

"jusqu'à 1 000" pour moi c'est qu'on ne renvoie jamais plus de 1000 résultats.